src/sql/src/session/vars.rs

```diff
@@ -2502,6 +2510,10 @@ impl SystemVars {
         *self.expect_value(&ENABLE_STORAGE_SHARD_FINALIZATION)
     }
 
+    pub fn enable_consolidate_after_union_negate(&self) -> bool {
```

Comment:
I thought this was added by the `feature_flags!` macro.

Reply:
In the final version I'm not using the `feature_flags!` macro, because it didn't seem possible to make it enabled by default using the macro. So I'm creating these things manually. Is that ok?

Reply:
Let me change the macro.

Reply:
Actually it you want to merge now just go for it. We can fix that later.

Reply:
Ok, thank you! Then let's fix the macro some other time. It's gonna be a bit tight already with running the mandatory nightly before the release cut.